atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/internalschema/TimestampPartitioningMap.java

```diff
@@ -153,18 +154,18 @@ public void check() {
         if (timestampRangeMap.asMapOfRanges().isEmpty()
                 || !timestampRangeMap.span().equals(ALL_TIMESTAMPS)) {
             throw new SafeIllegalArgumentException(
-                    "Attempted to initialize a timestamp partitioning map"
-                            + " of {}; its span does not cover precisely all timestamps.",
+                    "Attempted to initialize a timestamp partitioning map, but its span does not cover precisely all "
+                            + "timestamps.",
                     SafeArg.of("timestampToTransactionSchemaMap", timestampRangeMap));
         }
 
         RangeSet<Long> rangesCovered =
                 TreeRangeSet.create(timestampRangeMap.asMapOfRanges().keySet());
         if (rangesCovered.asRanges().size() != 1) {
             throw new SafeIllegalArgumentException(
-                    "Attempted to initialize a timestamp partitioning map"
-                            + " of {}. While the span covers all timestamps, some are missing. The disconnected ranges"
-                            + " of the provided map were {}.",
+                    "Attempted to initialize a timestamp partitioning map."
+                            + " While the span covers all timestamps, some are missing."
+                            + " The disconnected ranges of the provided map are below,",
```

Comment:
do note that they generally don't display with any guaranteed relative position internally, so `below` is a bit misleading!